### DISCUSSION TITLE
How I can convert batch .sfd files to .ttf files using the command line?

### DISCUSSION THREAD

**Discussion by xmha97** [fontforge/fontforge#4754] [Q&A] [answered]
Hi, I have a question:
How I can convert a `.sfd` file to a `.ttf` file using the command line?

**Comment by xmha97** [comment, marked as answer]
## Batch TTF to SFD
```powershell
Get-ChildItem -Filter '*.ttf' | % { &fontforge -lang=ff -c 'Open($1); Generate($2)' .\$($_.BaseName).ttf .\$($_.BaseName).sfd }
```
## Batch SFD to TTF
```powershell
Get-ChildItem -Filter '*.sfd' | % { &fontforge -lang=ff -c 'Open($1); Generate($2)' .\$($_.BaseName).sfd .\$($_.BaseName).ttf }
```

**Reply by ctrlcctrlv** [reply]
That's certainly not `bash`, `fish` or `zsh`. I guess PowerShell?

**Reply by ctrlcctrlv** [reply]
Uh huh, yeah, and scripts for it are usable on Linux too. Cool. Although the binary `pwsh` is huge, 170MB.

![image](https://user-images.githubusercontent.com/838783/121908513-153b5680-ccfb-11eb-85bf-230a266d797d.png)

**Reply by xmha97** [reply]
Yes, This's a PowerShell script.

**Reply by Lampe2020** [reply]
By the way, when has this been removed? I just tried it and it complained about the `-lang=ff`, about the `(` in the id (because it didn't even see that as a command) and then the ` ` in the id.
